src/main/java/uk/nhs/digital/nhsconnect/nhais/outbound/fhir/FhirController.java

```diff
@@ -34,10 +36,13 @@
     @PostMapping(path = "/fhir/Patient/{transactionTypeParam}", consumes = "application/json", produces = "application/json")
     @ResponseStatus(HttpStatus.ACCEPTED)
     public ResponseEntity<?> fromFhirToEdifact(@PathVariable String transactionTypeParam, @RequestBody String body) throws FhirValidationException {
+        LOGGER.info("Handling a request for Patient operation {}", transactionTypeParam);
         Parameters parameters = fhirParser.parseParameters(body);
         ReferenceTransactionType.Outbound transactionType = new TransactionTypeMapper().mapTransactionType(transactionTypeParam);
         OutboundMeshMessage meshMessage = fhirToEdifactService.convertToEdifact(parameters, transactionType);
+        LOGGER.info("Successfully translated transaction. OperationId={}", meshMessage.getOperationId());
```

Comment:
Consider adding OperationId to MDC and the log template

Reply:
Not going to do this because it is not present for much of the transaction. As long as its logged it can be linked the correlation id.